input/pagecontent/reference.md

```diff
@@ -2889,6 +2892,38 @@ A path rule has no impact on the element it refers to. The only purpose of the p
     * family MS
   ```
 
+{%include tu-div.html%}
+* Include optional fixed values of a path in an Instance:
+
+  Given a profile where name.given is optional and has a fixed value, such as:
+
+  ```
+  * name.given = "Doe"
+  ```
+
+  an Instance of that profile can include the fixed value "Doe" by including the following path rule:
+
+  ```
+  * name.given
+  ```
```

Comment:
I'd suggest using a different example. For one, it would be really weird to fix a given name in a profile. For two, who names their kid Doe?! ;-)

Might it also be helpful to show a case where a complex type is optional but has a fixed child element -- so by including a path to the parent, it triggers the whole type to be included with the required child?

Reply:
Whoops, that should have been `name.family`, but I'll update it regardless.

Reply:
I couldn't really think of a more realistic use case that someone would likely use, so I cleaned up this example a little in [e99b92b](https://github.com/HL7/fhir-shorthand/pull/152/commits/e99b92bc11d6fa53e8f05252ec3d1da98965fdc8). Let me know if you think there's another way to represent this idea that is more realistic.

Reply:
Looks good to me!